### ISSUE TITLE
Triple and quadruple dot Mathtext accents don't stack or align.

### ISSUE THREAD

**Issue by danzimmerman** [matplotlib/matplotlib#20595]
### Issue
Triple and quadruple dots accents used above a quantity for higher-order time derivatives do not align the same way as single and double: 

**Code**

```
plt.plot()
plt.gca().annotate(r'$\dot{\vec{q}}\hspace{1.5}\ddot{\vec{q}}\hspace{1.5}\dddot{\vec{q}}\hspace{1.5}\ddddot{\vec{q}}$', xy=(-0.025, 0))
```
**Expected Outcome**
The triple (`\dddot{\vec{q}}`) and quadruple (`\ddddot{\vec{q}}`) appear over the vector arrow like the single and double dots do 

**Actual Outcome**
The triple and quadruple dots align to the left of the vector arrow:

![image](https://user-images.githubusercontent.com/4365714/124831634-82808700-df49-11eb-82a9-f2f389d104a6.png)

Or whatever character is inside the brackets, here like `\dddot{q}`:

![image](https://user-images.githubusercontent.com/4365714/124835719-c4143080-df4f-11eb-9cff-622b7c859754.png)

I notice in the documentation that `\dddot` and `\ddddot` are [[not listed as accents]](https://matplotlib.org/stable/tutorials/text/mathtext.html#accents). Are they unsupported in mathtext? 

As a partial workaround, I can adjust the horizontal alignment with negative `\hspace` and switch to bolded vector notation but can't find something to adjust it vertically if I want to use the vector arrow:

![image](https://user-images.githubusercontent.com/4365714/124835491-697ad480-df4f-11eb-9687-a26e4cf77782.png)

**Version Info**

  * Operating system: Windows 11 Home Update 21H2
  * Matplotlib version: 3.4.2 (`conda-forge`)
  * Matplotlib backend: `module://ipykernel.pylab.backend_inline`
  * Python version: 3.8.8
  * Jupyter version (if applicable): Notebook version 6.3.0
  * Other Jupyter versions:
<details>

```

jupyter                   1.0.0            py38haa244fe_6    conda-forge
jupyter_client            6.1.12             pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.7.1            py38haa244fe_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_widgets        1.0.0              pyhd8ed1ab_1    conda-forge
```
</details>

**Comment by jklymak** [comment]
If its easy to add them, you probably could have a go at it.  I don't think those accents are heavily used, so there probably has not been demand to add them, versus any fundamental problem with them.

**Comment by QuLogic** [comment]
FWICT, the mapping to a glyph exists, but neither are in `_accent_map`, so they are not combined with any characters. There are also don't appear to be any glyph definitions for combining versions of those characters, so I'm not sure just adding to `_accent_map` would work without font support.

**Comment by danzimmerman** [comment]
> I'm not sure just adding to `_accent_map` would work without font support.

Yeah, it definitely "works on my machine" per the draft PR mentioned above but I don't know what else needs to happen to make sure it's fully supported in all relevant cases.

Does it also need entries in `latex_to_standard`?

**Comment by QuLogic** [comment]
Fixed by #20605, though there is an (unrelated) bug in the accent table, #20623.
